_sass/minima/_base.scss

```diff
@@ -130,6 +130,9 @@ li {
   }
 }
 
+li ul, li ol {
+  margin-top: 0;
```

Comment:
Did you validate across the whole site whether this messes up layouts anywhere? If this is meant only for main page content, it should go into `_article.scss`.

Reply:
I did, I scanned every page on the site. The only other place I found a nested list used was [on this page](https://deploy-preview-494--sad-borg-390916.netlify.app/guide/contribute/formatting/#nested-list) and I thought the change looked good there. How do you feel about it?